flashcards/backend/decks/admin.py

```diff
@@ -37,6 +37,28 @@ class DeckAdminConfig(admin.ModelAdmin):
     list_per_page = 50
 
 
+class FlashcardAdminConfig(admin.ModelAdmin):
+    """**FlashcardAdminConfig** is a configuration class for :class:`decks.models.Flashcard` model
+    in the admin panel."""
+
+    list_display = ("question", "answer", "rating_flashcard", "author", "is_active", "date_added")
+    search_fields = ("question",)
+    list_editable = ("is_active", "rating_flashcard")
+    list_display_links = ("question",)
+    save_on_top = True
+    list_filter = ("rating_flashcard", "is_active", "author", "category")
+    list_per_page = 50
+
+
+class DifficultyLevelAdminConfig(admin.ModelAdmin):
+    pass
```

Comment:
this, looks strange when you want to add, difficulty level in cms, probably we need to write model for difficulty level in another task

Reply:
@BartekSzymik this branch is not rebase on top of master. There is a Difficulty level model on master branch😉

Reply:
yes, of course there is, my mistake, sorry